src/components/forms/formUtils.js

```diff
@@ -0,0 +1,46 @@
+import {FormFieldTypes} from './forms';
+import { isNil, isFunction } from 'lodash/fp';
+
+const alwaysRequiredProps = ['id'];
+const alwaysPossibleProps = [
```

Comment:
what would you think of renaming this to `commonProps` or something instead? I'm imagining like


```suggestion
const commonProps = [
    id: { type: 'string', required: true },
    name: { type: 'string' },
    description: { type: 'string' },
    disabled: { type: 'bool' }
    ...
];
```

Reply:
sure! works for me.